okonomiyaki/utils/tests/test_test_data.py

```diff
@@ -0,0 +1,34 @@
+import sys
+import zipfile2
+
+from .. import test_data
+
+if sys.version_info < (2, 7):
+        import unittest2 as unittest
+else:
+        import unittest
+
+
+class TestDummyPythonRuntimes(unittest.TestCase):
+
+    def _get_contents_of_runtime(self, runtime):
+        with zipfile2.ZipFile(runtime) as zp:
+            return zp.namelist()
+
+    def test_pythonw_in_dummy_runtime(self):
+        """
+        Ensure that pythonw.exe is included in all Windows runtimes
+        """
+
+        # Given
+        win_runtimes = [f for f in dir(test_data)
```

Comment:
This is not the right way of detecting implementation/platform: you should instead use `okonomiyaki.runtimes.IRuntimeMetadata.factory_from_path`, which gives you an object w/ both a platform and implementation attributes.